src/OrchardCore/OrchardCore.DisplayManagement.Liquid/OrchardCoreBuilderExtensions.cs

```diff
@@ -79,6 +80,25 @@ public static OrchardCoreBuilder AddLiquidViews(this OrchardCoreBuilder builder)
                         };
                     });
 
+                    o.Scope.SetValue("Environment", new ObjectValue(new LiquidEnvironmentAccessor()));
+                    o.MemberAccessStrategy.Register<LiquidEnvironmentAccessor, FluidValue>((obj, name, ctx) =>
+                    {
+                        var hostEnvironment = ((LiquidTemplateContext)ctx).Services.GetRequiredService<IHostEnvironment>();
+
+                        if (hostEnvironment != null)
+                        {
+                            return name switch
+                            {
+                                "IsDevelopment" => BooleanValue.Create(hostEnvironment.IsDevelopment()),
+                                "IsStaging" => BooleanValue.Create(hostEnvironment.IsStaging()),
+                                "IsProduction" => BooleanValue.Create(hostEnvironment.IsProduction()),
```

Comment:
```suggestion
                                "IsProduction" => BooleanValue.Create(hostEnvironment.IsProduction()),
                                "EnvironmentName" => StringValue.Create(hostEnvironment.EnvironmentName),
```

Reply:
If the intend a custom environment name, I prefer `Name` instead because `Environment.Name` is make sense than `Environment.EnvironmentName`

Reply:
Yep, agree.